### DISCUSSION TITLE
desktop/web map download?

### DISCUSSION THREAD

**Discussion by nutpantz** [organicmaps/organicmaps#7638] [Ideas]
i update the maps on my devices for myself, and a number of friends. 
for osmand its easy, i download all the maps on my laptop and copy the files to everybody device from my laptop. ( 20 ish devices) 
but for organic maps there is no way to download maps from a webrowser that i know of so i just do it all on their devices when i see them over the month.   
  
its not a huge deal for me, but its a TON of bandwidth that could be saved on your end. ( 20 devices with about 1.5 gigs each)
and it would save each friend that data usage also.

**Comment by biodranik** [comment]
Did you check this? https://cdn.organicmaps.app/

**Reply by blockx** [reply]
That's a very useful link, thanks for providing that.  But I have a question in how to access the map files in OM after downloading them (which I have done successfully) ?  Is there some way to change the default web location that OM searched for updated maps ?

Thanks again,

**Reply by biodranik** [reply]
Check maps location in OM storage settings. You can adb push / adb pull them from external/SD storage.
